### PR TITLE
Fetch asset with ID 1 from the API and display it

### PR DESCRIPTION
**Pull request by danielburnley**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,9 +1,50 @@
...
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+import React from "react";
+import { mount } from "enzyme";
+import App from "./App";
+import GetAssetSimulator from "../test/Simulators/GetAsset";
+
+let asset = {
```
Comment (danielburnley) [inline src/App.test.js]: Probably can refactor building these into a more common place, cos they're massive

File: src/Components/AssetProvider/assetProvider.test.js [hunk 2]
```diff
@@ -1,127 +1,59 @@
...
+  describe("Example one", () => {
+    beforeEach(() => {
+      foundAsset = { cat: "meow" };
+      getAssetSpy = { execute: jest.fn(() => ({ asset: foundAsset })) };
+      childrenSpy = jest.fn();
 
-      shallow(
-        <AssetProvider searchAssets={searchAssetsSpy}>
-          {childrenFake.render}
+      mount(
```
Comment (PinkhamSoftware) [inline src/Components/AssetProvider/assetProvider.test.js]: What's this mount?

Comment (PinkhamSoftware) [reply src/Components/AssetProvider/assetProvider.test.js]: As discussed it's a way to render all components in Enzyme if I've got that correctly?

File: test/Components/Asset/index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import { mount } from 'enzyme'
+import Asset from '../../../src/Components/Asset'
+
+export default class AssetComponent {
+  constructor(assetData) {
+    this.component = mount(<Asset asset={assetData} />);
+  }
+
+  find(dataTest) {
+    return this.component.find({ "data-test": dataTest });
```
Comment (PinkhamSoftware) [inline test/Components/Asset/index.js]: The data-test was only relevant when testing things, as I recall.

File: src/Components/Asset/asset.test.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  agencyEquityLoan: 8765,
+  developerEquityLoan: 3219,
+  shareOfRestrictedEquity: 7654,
+  differenceFromImsExpectedCompletionToHopCompletionDate: 2198
+};
+
+describe("<Asset>", () => {
+  describe("Example one", () => {
+    it("Renders the asset details", () => {
+      let asset = new AssetComponent(assetOne);
```
Comment (PinkhamSoftware) [inline src/Components/Asset/asset.test.js]: Ok thanks for explaining the test data off the data-test properties.

**Pull request merged**